README.md

```diff
@@ -41,6 +41,25 @@ pip install tox
 tox
 ```
 
+### Docker Image
+You can also directly use TAPAS via Docker.
+
+For first time use of nvidia-docker, it is important to run the following commands:
+
+```
+distribution=$(. /etc/os-release;echo $ID$VERSION_ID)
+curl -s -L https://nvidia.github.io/nvidia-docker/gpgkey | sudo apt-key add -
+curl -s -L https://nvidia.github.io/nvidia-docker/$distribution/nvidia-docker.list | sudo tee /etc/apt/sources.list.d/nvidia-docker.list
```

Comment:
Is there and official documentation for this steps, it would be better to link directly there and keep the comment in the readme short.